tools/cdk-release/lib/private/print.ts

```diff
@@ -0,0 +1,23 @@
+import * as util from 'util';
+import { ReleaseOptions } from '../types';
+
+export function debug(opts: ReleaseOptions, message: string): void {
+  if (opts.verbose) {
+    // eslint-disable-next-line no-console
+    console.log(`[cdk-release] ${message}`);
+  }
+}
+
+export function debugObject(opts: ReleaseOptions, message: string, object: any): void {
+  if (opts.verbose) {
+    // eslint-disable-next-line no-console
+    console.log(`[cdk-release] ${message}:\n${JSON.stringify(object, null, 2)}`);
+  }
+}
+
+export function checkpoint(opts: ReleaseOptions, msg: string, args: any[]) {
```

Comment:
Not sure the word "checkpoint" is the right one. "checkpoint" is usually somewhat synonymous with "save point", "rollback point" etc.

Reply:
This was taken directly from `standard-version`. Changed to `notify()`, let me know if that name is better.